lizard_blockbox/views.py

```diff
@@ -759,7 +765,7 @@ def to_bookmark_url(self):
     @cached_property
     def breadcrumbs(self):
         result = super(SelectedMeasuresView, self).breadcrumbs
-        result.append(Action(name=self.page_title))
+        result.append(Action(name=SelectedMeasuresView.page_title))
```

Comment:
Deze snap ik niet, waarom niet self?

Reply:
Dat komt door de `super().breadcrumbs` erboven. in de oude situatie kreeg je als breadcrmbs:

'Automatische Import' - 'Automatische Import'.

Dit komt omdat `self` aan de `super` wordt meegegeven, waardoor in die super call, `self.page_title` hetzelfde blijft.

Ik denk dat ik de `breadcrumbs` ga hardcoden.